console/input.rst

```diff
@@ -221,7 +221,7 @@ There are five option variants you can use:
     Accept either the flag (e.g. ``--yell``) or its negation (e.g.
     ``--no-yell``).
 
-You can combine ``VALUE_IS_ARRAY`` with ``VALUE_REQUIRED`` or
```

Comment:
So the following would be correct?

You need to combine ``VALUE_IS_ARRAY`` with ``VALUE_REQUIRED`` or
 ``VALUE_OPTIONAL`` like this::

Reply:
What is confusing is that  'InputArgument::IS_ARRAY' and 'InputOption::VALUE_IS_ARRAY' can both used combined and for both the same description is used 'You can combine...', so automatically i thought they both share the same logic,
but only 'InputArgument::IS_ARRAY' can be used alone.